notebooks/deepmreye_example_usage_pretrained_weights.ipynb

```diff
@@ -9525,9 +9525,54 @@
    "source": [
     "---\n",
     "<a name=\"download\"></a>\n",
-    "## Download exemplary data\n",
+    "## Download exemplary data and pretrained model weights\n",
```

Comment:
To better view this notebook: https://github.com/Remi-Gau/DeepMReye/blob/test_demo/notebooks/deepmreye_example_usage_pretrained_weights.ipynb